app/src/main/java/com/example/linkdevmvvm/ui/homeNewsFeed/HomeNewsFeedFragment.java

```diff
@@ -0,0 +1,87 @@
+package com.example.linkdevmvvm.ui.homeNewsFeed;
+
+
+import android.arch.lifecycle.ViewModelProviders;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.util.Log;
+
+import com.android.databinding.library.baseAdapters.BR;
+import com.example.linkdevmvvm.R;
+import com.example.linkdevmvvm.adapters.NewsFeedAdapter;
+import com.example.linkdevmvvm.dataModel.Article;
+import com.example.linkdevmvvm.databinding.FragmentHomeNewsFeedBinding;
+import com.example.linkdevmvvm.ui.base.BaseFragment;
+import com.example.linkdevmvvm.ui.newsFeedDetails.NewsDetailsFragment;
+import com.example.linkdevmvvm.ui.newsFeedDetails.NewsFeedDetailsActivity;
+import com.example.linkdevmvvm.utils.Utils;
+
+
+public class HomeNewsFeedFragment extends BaseFragment<FragmentHomeNewsFeedBinding, HomeNewsFeedViewModel> implements NewsFeedAdapter.OnItemNewsClicked {
+
+    public static final String TAG = "HomeFragment";
+    FragmentHomeNewsFeedBinding fragmentHomeNewsFeedBinding;
+    private Context context;
+    private HomeNewsFeedViewModel homeNewsFeedViewModel;
+
+    public static HomeNewsFeedFragment getInstance() {
+        return new HomeNewsFeedFragment();
+    }
+
+    @Override
+    protected int getBindingVariable() {
+        return BR.viewModel;
+    }
+
+    @Override
+    protected int getLayoutId() {
+        return R.layout.fragment_home_news_feed;
+    }
+
+    @Override
+    protected HomeNewsFeedViewModel getViewModel() {
+        homeNewsFeedViewModel = ViewModelProviders.of(this).get(HomeNewsFeedViewModel.class);
+        return homeNewsFeedViewModel;
+    }
+
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        context = getActivity();
+        fragmentHomeNewsFeedBinding = getViewDataBinding();
+        handleNewsRequest();
+
+    }
+
+    private void handleNewsRequest() {
+        if (context == null)
+            return;
+        homeNewsFeedViewModel = ViewModelProviders.of(this).get(HomeNewsFeedViewModel.class);
+        homeNewsFeedViewModel.getNewsFeed(false);
+        homeNewsFeedViewModel.getHomeNewsResponse().observe(this, newsFeedResponse -> {
```

Comment:
You should replace lambda expression with method reference same as in NewsDetailsFragment